dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/websockets/HasWebSockets.scala

```diff
@@ -81,6 +81,46 @@ trait HasWebSockets extends BeforeAndAfterAll with BeforeAndAfterEach with HasJw
     c.clearMessages()
   }
 
+  protected def mergeWsOrder(orig: WsOrder, diff: WsOrder): WsOrder = WsOrder(
+    id = orig.id,
+    timestamp = diff.timestamp,
+    amountAsset = orig.amountAsset.orElse(diff.amountAsset),
+    priceAsset = orig.priceAsset.orElse(diff.priceAsset),
+    side = orig.side.orElse(diff.side),
+    isMarket = orig.isMarket.orElse(diff.isMarket),
+    price = orig.price.orElse(diff.price),
+    amount = orig.amount.orElse(diff.amount),
+    fee = orig.fee.orElse(diff.fee),
+    feeAsset = orig.feeAsset.orElse(diff.feeAsset),
+    status = orig.status.orElse(diff.status),
+    filledAmount = orig.filledAmount.orElse(diff.filledAmount),
+    filledFee = orig.filledFee.orElse(diff.filledFee),
+    avgWeighedPrice = orig.avgWeighedPrice.orElse(diff.avgWeighedPrice)
+  )
```

Comment:
This branch should be rebased from the master and a processing of new field is added.
WsOrder got a new field: https://github.com/wavesplatform/matcher/blob/master/dex/src/main/scala/com/wavesplatform/dex/api/ws/entities/WsOrder.scala#L29

Reply:
ok